theme-components/vertical-full-page-map/script.js

```diff
@@ -49,14 +49,11 @@ ANSWERS.addComponent('VerticalFullPageMapOrchestrator', Object.assign({},
       {{/if}}
     {{/each}}
   ],
-  searchbarConfig: Object.assign({}, 
```

Comment:
For my own edification, why do we no longer need the `searchbarConfig`?

Reply:
That was a residual from when we were trying out all components as child components of the orchestrator! That is no longer the case to keep the structure HHs are used to